### DISCUSSION TITLE
IStorageProvider API MVVM Scenario

### DISCUSSION THREAD

**Discussion by workgroupengineering** [AvaloniaUI/Avalonia#11093] [Q&A] [answered]
How to use IStorageProvider API in MVVM scenario?

**Comment by wieslawsoltes** [comment, marked as answer]
Just create service https://github.com/wieslawsoltes/ChatGPT/blob/2a13d32bb7255f761170091075d48a3e6010aa6f/src/ChatGPT.UI/Services/ApplicationService.cs#L87
https://github.com/wieslawsoltes/ChatGPT/blob/2a13d32bb7255f761170091075d48a3e6010aa6f/src/ChatGPT.Core/Model/Services/IApplicationService.cs#L8
